src/simulators/statevector/qubitvector_thrust.hpp

```diff
@@ -799,8 +799,8 @@ void QubitVectorChunkContainer<data_t>::SetupP2P(int nDev)
       for(i=0;i<nDev;i++){
         if(i != m_iDevice){
           cudaDeviceCanAccessPeer(&m_p2pEnable[i],m_iDevice,i);
-          if(m_p2pEnable[i])
-            cudaDeviceEnablePeerAccess(i,0);
+//          if(m_p2pEnable[i])
+//            cudaDeviceEnablePeerAccess(i,0);
```

Comment:
Is this change intended for this PR or was something done for testing?

Reply:
I found this will cause failure on unittest, so commented out